package/windows/pyinstaller.spec

```diff
@@ -147,20 +187,16 @@ def innosetup():
     subprocess.call(["iscc", os.path.join(SPECPATH, config_name)])
     os.remove(config_name)
 
-
-
-def make_zip():
-    """Create a zip archive of the fat binary files"""
-    from silx import strictversion
-
-    base_name = os.path.join(SPECPATH, "artifacts", f"silx-{strictversion}-windows-application")
+    # Create a zip archive of the fat binary files.

```

Comment:
Why not keeping the `innosetup` and `make_zip` helper functions instead of replacing them with a comment?

Reply:
I reverted to using functions. I thought it was more linear without them.